src/infohud.asm

```diff
@@ -312,29 +341,36 @@ ih_mb1_segment:
     ; runs during MB1 cutscene when you regain control of Samus, just before music change
     JSL $90F084    ; we overwrote this instruction to get here
 
-    JSL ih_update_hud_early
-    RTL
+    JML ih_update_hud_early
 }
 
 ih_mb2_segment:
 {
     ; runs during baby spawn routine for MB2
     STA $7E7854    ; we overwrote this instruction to get here
 
-    JSL ih_update_hud_early
-    RTL
+    JML ih_update_hud_early
 }
 
 ih_drops_segment:
 {
     ; runs when boss drops spawn
     JSL ih_update_hud_early
-    JSL $808111 ; overwritten code
-    RTL
+    JML $808111 ; overwritten code
+}
```

Comment:
Looks like the JML landed in the wrong spot on this one.

Reply:
Talked with Firebat and we think this is a non-issue.  I'm going to proceed with the merge.

It wouldn't surprise me to find an JML/JSL statement that is inaccurate for PAL; we still have more work to do for PAL.  For example, shinetune isn't up to date for PAL.